lib/webdriver.py

```diff
@@ -176,35 +228,37 @@ def _login_listener(self, data: JSON) -> None:
             logger.debug("Upcoming trips response has been received")
             self.trips_request_id = data["params"]["requestId"]
 
-    def _wait_for_attribute(self, attribute: str) -> None:
-        logger.debug("Waiting for %s to be set (timeout: %d seconds)", attribute, WAIT_TIMEOUT_SECS)
+    def _wait_for_attribute(
```

Comment:
As no calls to `_wait_for_attribute` are using a different timeout than `WAIT_TIMEOUT_SECS`, let's keep the original function header and remove the default timeout parameter.